routes.js

```diff
@@ -0,0 +1,91 @@
+const express = require("express")
+const router = express.Router()
+const low = require('lowdb');
+// const path = require('path')
+const fileAsync = require('lowdb/lib/storages/file-async');
+const db = low('db/db.json', {
+  storage: fileAsync
+});
+
+module.exports = router;
+
+router.get('/movies', function(req, res) {
+  res.send(db)
+})
+
+router.get('/movies/edit/:title', function(req, res) {
```

Comment:
You use the `:` in an interesting way, but it is worth pointing out that the `:` is only to tell express that this is a URL parameter and not a literal word to match. `/movies/edit/:title` will take whatever follows `/movies/edit/` and throw it into `req.params.title`. A `:` is not needed in the actual path in Chrome to work.